multicluster-config/observer-values.yaml

```diff
@@ -52,7 +52,19 @@ thanos:
   existingObjstoreSecret: thanos-objstore-config
   queryFrontend:
     enabled: true
+    extraFlags:
```

Comment:
IMO this file should be used as a values file which should override the default values file: 
- `helm install .... -f multicluster-config/observer-values.yaml.yaml` 

then I do not think that these changes are necessery